Common/API/LiveAlgorithmResults.cs

```diff
@@ -0,0 +1,208 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using QuantConnect.Api;
+using QuantConnect.Orders;
+
+namespace QuantConnect.API
+{
+    /// <summary>
+    /// Details a live algorithm from the "live/read" Api endpoint
+    /// </summary>
+    public class LiveAlgorithmResults : RestResponse
+    {
+        /// <summary>
+        /// Represents data about the live running algorithm returned from the server
+        /// </summary>
+        public LiveResults LiveResults { get; set; }
+    }
+
+    /// <summary>
+    /// Holds information about the state and operation of the live running algorithm
+    /// </summary>
+    public class LiveResults
```

Comment:
The nulls are to demark the start and endpoint of various deployments of the same algorithm. The null breaks the line so its not a constant and looks more like a dash "----   --  ---     -" etc.

Probably best we'll need a nullable int (wide spread impact) or custom deserializer (localized impact) for this like how we do custom Order type deserialization. Please see how wide the impact would be setting it to a nullable int & we can decide the right path. Setting to -1 or 0 would throw chart scales off; so the only other option would be to skip the point. 

Wishing we'd made ChartPoint a class now :) Then we could set the whole point to null.